credoai/prism/comparators/metric_comparator.py

```diff
@@ -0,0 +1,106 @@
+"""Comparators for metric type containers"""
+from typing import Callable, List, Literal, Optional
+
+import numpy as np
+from pandas import DataFrame, concat
```

Comment:
Interestingly I always thought referencing only the method you need was preferable, mainly based on the Zen of Python: "Explicit is better than implicit". Also if the imported package is big, it makes evident which are the parts we depend on.

Why was your preference for the `import package as ...` syntax ?

Reply:
My preference for `import package as` is because I find it _more explicit_. We import the entire package regardless and you can easily see that this is "pd.DataFrame", rather than some other DataFrame. It's also the convention I see everywhere with pandas and numpy.